build.xml

```diff
@@ -390,6 +390,48 @@
         </antcall>
     </target>
 
+    <target name="compare-versions" description="Compares the canonicalized sources of the MEI dev version with the previous stable version and creates an HTML output; custom versions and output folder can be set via -Dsource, -Dold and -Doutput input params. NB: path of old file is relative to comparison.xml, while others are relative to build.xml." depends="canonicalize-source">
+        <!-- set source file (relative to build.xml) -->
+        <property name="source.default" value="${dir.build}/mei-source_canonicalized.xml"/>
+        <echo level="verbose">source.default: ${source.default}</echo>
+        <echo level="verbose">source user input: ${source}</echo>
+
+        <condition property="source.file" value="${source}" else="${source.default}">
+            <isset property="source"/>
+        </condition>
+        <echo>Using source.file: ${source.file}</echo>
+
+        <!-- set old file to compare with (relative to comparison.xml) -->
+        <property name="old.default" value="mei-source_canonicalized_v4.0.1.xml"/>
```

Comment:
Why do you want this relative to `comparison.xsl`? The ant file will treat such paths relative to itself. So in terms of, e.g. checking the availability of $old file, it would make more sense to make this relative to build.xml or silently add the folder to a check (cf. https://github.com/musicEnfanthen/music-encoding/pull/1)

Reply:
It was relative to comparison.xsl because the XSLT did not find it otherwise. But I think it should work with absolute path.